### ISSUE TITLE
make a 1.0 release?

### ISSUE THREAD

**Issue by orbeckst** [datreant/MDSynthesis#64]
The docs still state that this is alpha software but I think in truth it has been in stable use for more than a year. Isn't it time to slap a 1.0 on it?

(Or are we waiting for MDAnalysis 0.16.0?)

**Comment by kain88-de** [comment]
I would say after this datreant issue https://github.com/datreant/datreant.core/issues/100 has been resolved we can go for a 1.0 release. Solving that issue will mean that old sim files need to be changed by the user to still function.

**Comment by orbeckst** [comment]
ping @dotsdl – ready to :ship: ?

**Comment by kain88-de** [comment]
ping @dotsdl 

anything holding this back? The new datreant libraries are shipped and the develop branch is tested well

**Comment by dotsdl** [comment]
@kain88-de only the docs. Hacking on this now.

**Comment by dotsdl** [comment]
Just realized that we have to graft `datreant.data.agglimbs.AggData` onto `Bundle` and `View`, as well as `datreant.data.limbs.Data` onto `Tree` if we want to maintain the full `data` functionality. We do this either with subclassing or monkey-patching to the classes, neither of which is very satisfying to me. I'm tempted to ditch `data` entirely, but that's easy for me to say because I don't myself depend on it.

@kain88-de, thoughts? This whole library is a bit of a shim in my view, so I'm not inclined to expend too much effort to keep it all running.

**Comment by kain88-de** [comment]
I thought we have most of the data functionality ported into mdsynthesis by now? The `AggData` is to work on a bundle?

**Comment by dotsdl** [comment]
Yeah, `AggData` is for `Bundle`s (and `View`s), while `Data` is for `Treant`s (and `Tree`s). I think what I'll do is subclass `Bundle` from `datreant` and graft `AggData` onto it, then we'll cut our losses and forget trying to get these limbs to work for `View` and `Tree`, since that takes us down a rabbit hole. Subclassing `Bundle` is already well-motivated since we want the MDS `Bundle` to handle `Sim`s and only `Sim`s.

**Comment by kain88-de** [comment]
Yeah. Just getting bundle to work is enough. I have never really used this
feature myself.

My time to work on this for the next 6 months is unfortunately 0.

**Comment by dotsdl** [comment]
No worries, I can handle this. I often get stuck in the mud when I know what my needs are (*not MDSynthesis*), but don't know what users expect to keep working. I'll proceed with the path I laid out, and trim the docs down accordingly.

**Comment by kaceyaurum** [comment]
I love mdsynthesis; make a 1.0 already! As long as the basics (bundles!) work, people will be happy.

**Comment by orbeckst** [comment]
A 1.0 would be nice – then it's done. conda installs MDS 0.6.1, which seems to be happy with datreant 1.0 – I hope this works...
